### PR TITLE
lsfd: add page

### PR DESCRIPTION
**Pull request by Managor**
#11827

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/lsfd.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+`lsfd`
+
+- List all files kept open by a specific program:
+
+`lsfd -Q 'PID == {{process_ID}}'`
+
+- Check what program has a specific file open:
+
+`lsfd -Q "NAME == '/path/to/file'"`
```
Comment (gutjuri) [inline pages/linux/lsfd.md:16]:
```suggestion
`lsfd -Q "NAME == '{{path/to/file}}'"`
```

**Review by gutjuri — APPROVED** [review]

File: pages/linux/lsfd.md (hunk 1 above)
Comment (Managor) [reply pages/linux/lsfd.md:16]: Absolute path is a must

Comment (gutjuri) [reply pages/linux/lsfd.md:16]:
```suggestion
`lsfd -Q "NAME == '{{/path/to/file}}'"`
```

Okay, I still think that we should add a placeholder here.

File: pages/linux/lsfd.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# lsfd
+
+> Lists open files and the corresponding processes in Linux.
```
Comment (kbdharun) [inline pages/linux/lsfd.md:3]:
```suggestion
> List open files and the corresponding processes in Linux.
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**